packages/cli/src/commands/generate/scaffold/templates/components/NameForm.tsx.template

```diff
@@ -27,21 +42,60 @@ const ${singularPascalName}Form = (props) => {
           titleClassName="rw-form-error-title"
           listClassName="rw-form-error-list"
         />
-<% editableColumns.forEach(column => { %>
+      <% editableColumns.forEach(column => { %>
         <Label
           name="${column.name}"
           className="rw-label"
           errorClassName="rw-label rw-label-error"
         >
           ${column.label}
         </Label>
-        <${column.component}
-          name="${column.name}"
-          ${column.defaultProp}={${column.deserializeFunction && column.deserializeFunction + '('}props.${singularCamelName}?.<%= column.name %>${column.deserializeFunction && ')'}}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"<%= !column.validation ? '' : `
-          validation=${column.validation}` %>
-        />
+
+        <% if (column.isEnum) { %>
+
+          <% if (!column.isRequired) { %>
+            <div className="rw-check-radio-items">
+              <${column.component}
+                id="${singularCamelName}-${column.name}-none"
+                name="${column.name}"
+                defaultValue=""
+                ${column.defaultProp}={!props.spot?.spotType}
+                className="rw-input"
+                errorClassName="rw-input rw-input-error"
+              />
+              <div>
+                None
```

Comment:
What do you think about adding some style to this to make it italic? That'd make *None* stand out a bit to show that it's a special value

Reply:
Not sure I love this:

![image](https://user-images.githubusercontent.com/1051633/153081603-6050bf47-0c47-4abe-a9b7-86d7d9779380.png)

Reply:
Sure. Any other ideas? What about explaining it some more? "* None (will not set this value in the database)". Ideally something a bit shorter than that though 😅

Reply:
Maybe just "* Not set" is enough? Could also play around with text color to signify that it's a special value